### DISCUSSION TITLE
discordpy

### DISCUSSION THREAD

**Discussion by tachfinedxs** [Rapptz/discord.py#9199] [Q&A] [answered]
m trying to make a discord bot but when  i try to run the file it says 
**_missing 1 required keyword-only arguments : 'intentes'_**
pls help

**Comment by Soheab** [comment, marked as answer]
See this https://discordpy.readthedocs.io/en/latest/migrating.html#intents-are-now-required and this https://discordpy.readthedocs.io/en/latest/intents.html
